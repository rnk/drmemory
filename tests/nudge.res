# **********************************************************
# Copyright (c) 2011 Google, Inc.  All rights reserved.
# Copyright (c) 2009-2010 VMware, Inc.  All rights reserved.
# **********************************************************
#
# Dr. Memory: the memory debugger
#
# This library is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation; 
# version 2.1 of the License, and no later version.
#
# This library is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Library General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public
# License along with this library; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
#
%if WINDOWS
# XXX: extra leak due to encoded pointer (PR 482555) is no longer happening on
# my machine!  not sure what's going on
#Error #1: LEAK 128 direct bytes + 0 indirect bytes
#crtheap.c:61
#Error #2: LEAK 160 direct bytes + 0 indirect bytes
#infloop.c:91
#Error #3: LEAK 42 direct bytes + 17 indirect bytes
#infloop.c:80
Error #1: LEAK 160 direct bytes + 0 indirect bytes
infloop.c:92
Error #2: LEAK 42 direct bytes + 17 indirect bytes
infloop.c:81
%endif
%OUT_OF_ORDER
%if UNIX
LEAK 160 direct bytes + 0 indirect bytes
infloop.c:92
LEAK 42 direct bytes + 17 indirect bytes
infloop.c:81
%endif
