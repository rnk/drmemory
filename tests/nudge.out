# **********************************************************
# Copyright (c) 2011 Google, Inc.  All rights reserved.
# Copyright (c) 2009-2010 VMware, Inc.  All rights reserved.
# **********************************************************
#
# Dr. Memory: the memory debugger
#
# This library is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation; 
# version 2.1 of the License, and no later version.
#
# This library is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Library General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public
# License along with this library; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
#
starting
~~Dr.M~~ ERRORS FOUND:
~~Dr.M~~       0 unique,     0 total unaddressable access(es)
~~Dr.M~~       0 unique,     0 total uninitialized access(es)
~~Dr.M~~       0 unique,     0 total invalid heap argument(s)
~~Dr.M~~       0 unique,     0 total warning(s)
%if WINDOWS
# XXX: extra leak due to encoded pointer (PR 482555) is no longer
# happening on my machine!  not sure what's going on
#~~Dr.M~~       3 unique,    22 total,   3387 byte(s) of leak(s)
~~Dr.M~~       2 unique,    21 total,   3259 byte(s) of leak(s)
# FIXME PR 485354: sometimes some or all of the leaks are "possible": why?
~~Dr.M~~       0 unique,     0 total,      0 byte(s) of possible leak(s)
%endif
%if UNIX
~~Dr.M~~       2 unique,    21 total,   3259 byte(s) of leak(s)
~~Dr.M~~       0 unique,     0 total,      0 byte(s) of possible leak(s)
%endif
